### ISSUE TITLE
Add keyboard shortcut to answer nextcloud talk call on PC

### ISSUE THREAD

**Issue by grishakonzalaev** [nextcloud/talk-desktop#1684]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
Blind users cannot answer nextcloud calls.
**Describe the solution you'd like**
Add global keyboard shortcut to answer calls.
**Describe alternatives you've considered**

**Additional context**

**Comment by grishakonzalaev** [comment]
Screen reader don't focus in popup window if ringing. I Use NVDA screen reader. I can view this popup only when use OCR, but it's very slow to answer call.
System alerts don't shows in win11 when ringing. Only when call missed.

**Comment by ShGKme** [comment]
I'm not sure about a global shortcut. It is always problematic and needs to avoid conflicts with other apps in the global namespace.

We don't put the focus on the call popup intentionally. To avoid an accident, accepting/rejecting a call when the popup pops up during typing, for example.

System alert can be enabled as a replacement for the callbox popup as a replacement in the app settings.

We'll have a look at common patterns for this in other meeting software.

**Comment by grishakonzalaev** [comment]
Thanks! It works! but it would be more convenient to make optional global shortcut to answer call, because it need two action^ 1. Press to system alert and 2. press join call.
